platform/src/event/finger.rs

```diff
@@ -1064,7 +1143,7 @@ impl Event {
                 if cx.fingers.test_sweep_lock(options.sweep_area) {
                     return Hit::Nothing;
                 }
-                let device = DigitDevice::Mouse { button: 0 };
+                let device = DigitDevice::Mouse { button: MouseButton::empty() };
```

Comment:
@makepaddev is this correct? Previously, a button value of `0` meant the primary button, but that seems weird to me. Thus, I used a "no" button bitmask instead, which indicates that no button was pressed.

I'm not quite sure why a `MouseLeave` event needs a button press, but do let me know if we need to switch it to `MouseButton::PRIMARY`.